### ISSUE TITLE
[Build] Openvino debug build fails on VS2019

### ISSUE THREAD

**Issue by BengtGustafsson** [microsoft/onnxruntime#15496]
### Describe the issue

I get the following build errors with openvino when building in debug mode on Windows64, VS 2019.

I'm building branch "main" as tag 1.14 can't build Win32 dml. The hash when this occurred was: d49a8de9b1

### Target platform

Win64

### Build script

2023-04-13 13:03:16,283 build [DEBUG] - Command line arguments:

  --build_dir 'j:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\\build\Windows' --config Debug --parallel 4 --build_shared_lib --build_dir build_gpu/Windows --use_cuda --use_dml --use_openvino --cuda_version=11.5 --cuda_home 'C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA' --cudnn_home 'C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA'

"C:\Program Files\CMake\bin\cmake.EXE" --build "build_gpu/Windows\Debug" --config Debug -- /maxcpucount:4 /nodeReuse:False /p:CL_MPCount=4

### Error / output

J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\onnxruntime\core\providers\openvino\backend_utils.cc(102,17): error C2039: 'Serialize': is not a member of 'ov::pass' [J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\build_gpu\Windows\Debug\onnxruntime_providers_openvino.vcxproj]
C:\Program Files (x86)\Intel\openvino_2022\runtime\include\openvino/pass/constant_folding.hpp(11): message : see declaration of 'ov::pass' [J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\build_gpu\Windows\Debug\onnxruntime_providers_openvino.vcxproj]
J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\onnxruntime\core\providers\openvino\backend_utils.cc(102,27): error C2065: 'Serialize': undeclared identifier [J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\build_gpu\Windows\Debug\onnxruntime_providers_openvino.vcxproj]
J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\onnxruntime\core\providers\openvino\backend_utils.cc(102,27): error C2146: syntax error: missing ';' before identifier 'serializer' [J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\build_gpu\Windows\Debug\onnxruntime_providers_openvino.vcxproj]
J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\onnxruntime\core\providers\openvino\backend_utils.cc(102,27): error C3861: 'serializer': identifier not found [J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\build_gpu\Windows\Debug\onnxruntime_providers_openvino.vcxproj]
J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\onnxruntime\core\providers\openvino\backend_utils.cc(103,7): error C2065: 'serializer': undeclared identifier [J:\workspace\Bitbucket_thirdpartylibs_PR-15@2\win64\onnxruntime\build_gpu\Windows\Debug\onnxruntime_providers_openvino.vcxproj]

### Visual Studio Version

VS2019

**Comment by BengtGustafsson** [comment]
I don't know which version we have. Will check. But note, in your doc. it says 2022.2 should be ok:
https://onnxruntime.ai/docs/build/eps.html#openvino

Also note that release build works. I've had problems before with building in debug, so maybe you don't do that in CI before merging?

**Comment by sfatimar** [comment]
Yes we have issues with OV Execution Provider Debug Build on Windows

**Comment by sfatimar** [comment]
But on Linux it is possible to build OVEP Debug Build

**Comment by sfatimar** [comment]
Also for serializer API, I think you should test now on main as were putting in fixes and that might have broken backward compatibility
